iOS/MaterialComponents/source/MaterialComponents/Extensions.cs

```diff
@@ -35,20 +113,22 @@ public static class MaterialFeatureHighlightStrings {
 	}
 
 	public partial class CardScheme {
-		SemanticColorScheme semanticColorScheme;
 		public SemanticColorScheme SemanticColorScheme {
 			get {
-				if (semanticColorScheme == null)
-					semanticColorScheme = Runtime.GetNSObject<SemanticColorScheme> (ColorScheme.Handle, false);
-
-				return semanticColorScheme;
+				return Runtime.GetNSObject<SemanticColorScheme> (ColorScheme.Handle, false);
```

Comment:
I may be wrong, but we shouldn't cache this as changing the other property will break this? This is supposed to be the same object - just a more type-specific version. ie: SemanticColorScheme === ColorScheme